### ISSUE TITLE
Zoom level proportional to deltaY on mouse wheel event

### ISSUE THREAD

**Issue by Jiogo18** [microsoft/vscode#157299]
## The feature
The Zoom level with ctrl + scroll is proportional to deltaY (times a sensitivity setting or just a constant).
This change can be done in mouseHandler.ts with something such as
```ts
const zoomFactor: number = this._context.configuration.options.get(EditorOption.mouseWheelZoomFactor); // default: 0.01 ?
const delta = e.deltaY * zoomFactor;
```
https://github.com/microsoft/vscode/blob/c3d9e2253a6fda9a938aa5be9c45714c66b9e168/src/vs/editor/browser/controller/mouseHandler.ts#L141

## Why ?
Touchpad events have a small deltaY compared to a physical wheel event, which is offset by more events:
|Wheel    |deltaY| number of events to scroll by 1 page|
|----------|-------|-------------------------|
|Physical  |  100  |             ~9                     |
|TouchPad|  5     |            ~220                  |

When Ctrl is pressed, the zoom level only depends on the sign of deltaY. The more events, the more it zooms.
For a TouchPad, I find it too sensitive.

**Comment by carvalhopatrick** [comment]
Ideally, it would be great to be able to distinguish between an scroll event coming from touchpad and it coming from a mouse whell, so this could be applied only for the touchpad zoom, and also fix issues like #147851. 

But it doesn't look like it's possible, as alexdima pointed on that issue. 
Still, this zoomFactor seems like an easy implementation, and one could change the value easily when switching from mouse/touchpad using extensions such as [Toggle](https://marketplace.visualstudio.com/items?itemName=rebornix.toggle)

**Comment by alexdima** [comment]
Let's track in #72050
